modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/Views.tsx

```diff
@@ -34,17 +34,11 @@ export default function Views({
 	style,
 	url,
 }: IFDSTableProps) {
-	const [creationLanguageId, setCreationLanguageId] = useState<
-		Liferay.Language.Locale
-	>();
-
 	useEffect(() => {
```

Comment:
Hi @juanjofgliferay. We can remove this useEffect since it was only used the API's response to set the creationLanguageId.